app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -85,6 +85,11 @@ class PasswordFragment : Fragment(R.layout.password_recycler_view) {
                     startActivityForResult(intent, operationId)
                 }
             }
+        } else {
```

Comment:
I don't know about all the workflows that allow to switch between a Git and a non-Git repository, but perhaps it would be safer to move the `if` into the `OnRefreshListener`?